setup.py

```diff
@@ -62,7 +62,7 @@ def build_extension(self, ext):
     author_email='angus@agibson.me',
     description='A python wrapper for the ROL package.',
     long_description='',
-    ext_modules=[CMakeExtension('ROL')],
+    ext_modules=[CMakeExtension('_ROL')],
```

Comment:
I guess this is necessary? I don't remember why I did it the not-underscore way in the first place!